src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -1985,6 +1989,11 @@ namespace GraphQL.Types
         GraphQL.Language.AST.IValue Convert(object value, GraphQL.Types.IGraphType type);
         bool Matches(object value, GraphQL.Types.IGraphType type);
     }
+    public interface ICanParseScalar
+    {
+        bool CanParseLiteral(GraphQL.Language.AST.IValue value);
+        bool CanParseValue(object value);
```

Comment:
I see the project is not using `CanParseValue` just now.  I think this is fine, and I do not think we should make any other changes to variable parsing to use it.

Reply:
So? Let new method be or remove?

Reply:
I suggest we leave it there.  We may find a use for it later, and it matches `CanParseLiteral`.

Reply:
Agree.